deploy/install.sh

```diff
@@ -93,7 +94,32 @@ applyDynatraceOperator() {
 }
 
 applyDynaKubeCR() {
-  cat <<EOF | "${CLI}" apply -f -
+  if [ -n "$CLUSTER_NAME" ]; then
```

Comment:
If I'm not mistaken, `-n` means non-empty, and this example doesn't use `CLUSTER_NAME`, so perhaps it's the other way around?